it/config/DocumentationISpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package config
+
+import play.api.http.Status
+import play.api.libs.json.Json
+import play.api.libs.ws.WSResponse
+import support.IntegrationBaseSpec
+
+class DocumentationISpec extends IntegrationBaseSpec {
+
+  val apiDefinitionJson = Json.parse(
+    """
+      |{
+      |  "scopes":[
+      |    {
+      |      "key":"read:self-assessment",
+      |      "name":"#name of scope#",
+      |      "description":"#Description of scope#"
+      |    },
+      |    {
+      |      "key":"write:self-assessment",
+      |      "name":"#name of scope#",
+      |      "description":"#Description of scope#"
+      |    }
+      |  ],
+      |  "api":{
+      |    "name":"#mtd-api# (MTD)",
+      |    "description":"#Description of API#,
+      |    "context":"mtd/template",
+      |    "versions":[
+      |      {
+      |        "version":"1.0",
+      |        "status":"ALPHA",
+      |        "endpointsEnabled":true
+      |      }
+      |    ]
+      |  }
+      |}
+    """.stripMargin)
+
+  "GET /api/definition" should {
```

Comment:
This suite of tests aborts for me. Not sure why. Does it pass for you?

Reply:
No. But this was kind of deliberate to indicate that the definition fields needed to be set to something.

Reply:
Oh okay, that's good to know 🙂